subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/versions/KotlinGradlePluginVersions.groovy

```diff
@@ -23,7 +23,7 @@ class KotlinGradlePluginVersions {
 
     // https://search.maven.org/search?q=g:org.jetbrains.kotlin%20AND%20a:kotlin-project&core=gav
     private static final List<String> LATEST_VERSIONS = [
-        '1.3.21', '1.3.31', '1.3.41', '1.3.50', '1.3.61', '1.3.72',
```

Comment:
I think all people should be using the latest patch version, wdyt? @eskatos

Reply:
Or "selectively", do you mean like `1.3.21`/`1.3.41`/`1.3.72`? I don't think it makes much sense to test old patch version.

Reply:
Note that in Kotlin terms, 1.3.72 is a patch release for the incremental release 1.3.70.
See https://kotlinlang.org/releases.html

Changing the minimal supported Kotlin version to 1.3.72 in Gradle 7.0 and keeping coverage for all incremental releases of the 1.4.x train makes sense to me :+1: